de-lib/src/main/java/org/iplantc/de/admin/desktop/client/services/impl/AppAdminServiceFacadeImpl.java

```diff
@@ -110,18 +111,12 @@ public void deleteApp(final HasQualifiedId app,
     }
 
     @Override
-    public void getAppDetails(final HasQualifiedId app,
-                              final AsyncCallback<App> callback) {
-        String address = APPS_ADMIN + "/" + app.getSystemId() + "/" + app.getId() + "/details";
-
+    public void getAppDetails(final String appId,
+                              final String systemId,
+                              final AsyncCallback<Splittable> callback) {
+        String address = APPS_ADMIN + "/" + systemId + "/" + appId + "/details";
         ServiceCallWrapper wrapper = new ServiceCallWrapper(GET, address);
-        deService.getServiceData(wrapper, new AsyncCallbackConverter<String, App>(callback) {
-            @Override
-            protected App convertFrom(String object) {
-                AutoBean<App> appAutoBean = AutoBeanCodex.decode(factory, App.class, object);
-                return appAutoBean.as();
-            }
-        });
+        deService.getServiceData(wrapper, new SplittableCallbackConverter(callback));
```

Comment:
Since Admin app details view already in react, I updated this service to return splittable.